### DISCUSSION TITLE
Team Fortress 2

### DISCUSSION THREAD

**Discussion by CallMeEchoCodes** [catppuccin/catppuccin#2053] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

TF2 has a robust modding system and ive seen many other HUD mods that completely remake the HUD textures and layout. I would probably keep it vanilla-ish with a more flat design to fit the catppuccin aesthetic better.  

### Any additional comments?

I know nothing about TF2 modding. I can learn but i will probably need someone in the community to help me out. If you know anything about source engine modding please ping me in the discord :)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by leofleeo** [comment]
Personally I think the hypnotize hud would be a good starting point. It has colors that are easily customizable in the code and it is material design inspired. https://github.com/Hypnootize/hypnotizehud

**Comment by leofleeo** [comment]
It's probably too late now so whatever

**Comment by PoorPocketsMcNewHold** [comment]
Could also just be different sets of colors combination. Some already done a couple for ToonHUD for example. 
https://toonhud.com/user/xoinks/theme/VATE4JGS/
https://toonhud.com/user/lemonaids/theme/TC1B6K9F/
https://toonhud.com/user/probroj/theme/D57D3EEX/
